gnucash/html/gnc-html-webkit2.c

```diff
@@ -52,12 +52,8 @@
 #include "gnc-prefs.h"
 #include "gnc-gui-query.h"
 #include "gnc-engine.h"
-#include "gnc-html.h"
 #include "gnc-html-webkit.h"
 #include "gnc-html-history.h"
-#include "print-session.h"
-#include "gnc-state.h"
-#include "print-session.h"
```

Comment:
These aren't variables, nor does clang-tidy flag them as unused. We want to follow [include-what-you-use](https://google.github.io/styleguide/cppguide.html#Include_What_You_Use) so even if they are included via some other  include they still need to be here.

Reply:
clang-tidy flags them all as unused here (15.0.2 on ubuntu 22.10)
`#include "gnc-state.h"` has to go because it is no longer being used (the call to gnc_state_get_current() is being removed)
The duplicate `#include "print-session.h"` is clearly wrong, but I can leave it if you want

Reply:
I checked manually. gnc-state.h's only use is as you say also removed in this change. print-session.h isn't, but gnc-html.h declares several macros and functions used in this file so it should stay.